src/main/java/mate.academy/lib/Component.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.lib;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
 public @interface Component {
```

Comment:
The `@Component` annotation is missing the `@Target` annotation. According to best practices, you should specify where this annotation can be applied using `@Target`. Consider adding it to improve the clarity and correctness of your annotation.